docs/docsite/rst/dev_guide/testing/sanity/import.rst

```diff
@@ -1,5 +1,69 @@
 import
 ======
 
-All Python imports in ``lib/ansible/modules/`` and ``lib/ansible/module_utils/`` which are not from the Python standard library
-must be imported in a try/except ImportError block.
+All Python imports in the directories specified that are not from a limited set of possibilities listed below
+must be imported in a try/except ImportError block as follows:
+
+1. In modules:
+
+   .. code-block:: python
+
+       # Instead of 'import another_library', do:
+
+       import traceback
+
+       try:
+           import another_library
+       except ImportError:
+           HAS_ANOTHER_LIBRARY = False
+           ANOTHER_LIBRARY_IMPORT_ERROR = traceback.format_exc()
+       else:
+           HAS_ANOTHER_LIBRARY = True
+
+
+       # Later in module code:
+
+       module = AnsibleModule(...)
+
+       if not HAS_ANOTHER_LIBRARY:
+           # Needs: from ansible.module_utils.basic import missing_required_lib
+           module.fail_json(
+               msg=missing_required_lib('another_library'),
+               exception=ANOTHER_LIBRARY_IMPORT_ERROR)
+
+2. In plugins:
+
+   .. code-block:: python
+
+       # Instead of 'import another_library', do:
+
+       from ansible.module_utils.six import raise_from
+
+       try:
+           import another_library
+       except ImportError as imp_exc:
+           ANOTHER_LIBRARY_IMPORT_ERROR = imp_exc
+       else:
+           ANOTHER_LIBRARY_IMPORT_ERROR = None
+
+
+       # Later in plugin code:
```

Comment:
```suggestion
       # Later in plugin code:
```
let's specify a suitable place maybe, e.g. in `__init__`?:)

Reply:
@Andersson007 how about:
```suggestion
       # Later in plugin code, for example in __init__ of the plugin:
```